docs/apis/streaming/index.md

```diff
@@ -1357,20 +1364,27 @@ Custom:
 
 <br />
 
-Sources can by created by using `StreamExecutionEnvironment.addSource(sourceFunction)`.
-You can either use one of the source functions that come with Flink or write a custom source
-by implementing the `SourceFunction` for non-parallel sources, or by implementing the
-`ParallelSourceFunction` interface or extending `RichParallelSourceFunction` for parallel sources.
+Sources are where your program reads its input from. You can attach a source to your program by 
+using `StreamExecutionEnvironment.addSource(sourceFunction)`. Flink comes with a number of pre-implemented 
+source functions, but you can always write your own custom sources by implementing the `SourceFunction` 
+for non-parallel sources, or by implementing the `ParallelSourceFunction` interface or extending the 
+`RichParallelSourceFunction` for parallel sources.
 
 There are several predefined stream sources accessible from the `StreamExecutionEnvironment`:
 
 File-based:
 
-- `readTextFile(path)` / `TextInputFormat` - Reads files line wise and returns them as Strings.
```

Comment:
It seems that the Java / Scala variants of the documentation are the same. I think we should remove the tab and show it once.